css-canvas-medical/styles.css

```diff
@@ -0,0 +1,221 @@
+body {
+  width: 80%;
+  margin: auto;
+}
+
+button {
+  border-radius: 0.5rem;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-half {
+  width: 50%;
+}
+
+.heading-text-right {
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+  top: 50%;
+  font-family: 'Avenir', sans-serif;
+  margin-right: 5rem;
+}
+
+.heading-text-left {
+  display: flex;
+  font-size: 20pt;
+  font-family: 'Tiempos Headline', sans-serif;
+  color: #0b4265;
+}
+
+.contact-us-button {
+  display: flex;
+  position: relative;
+  bottom: 10px;
+}
+
+.contact-us-button > button {
+  padding: 10px;
+  background-color: #294464;
+  color: white;
+  font-family: 'Avenir', sans-serif;
+}
+
+.hero-img {
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.hero-img > img {
+  width: 100%;
+  filter: brightness(75%);
+  padding: 5rem;
+}
+
+.hero-img-text {
+  display: flex;
+  flex-wrap: wrap;
+  color: #fff;
+  font-family: 'Avenir', sans-serif;
+  position: absolute;
+  padding-left: 5rem;
+  padding-top: 7rem;
+  top: -1rem;
+  line-height: 2rem;
+  width: 40rem;
+  left: 7rem;
+}
+
+.hero-img-text > h1 {
+  font-size: 32pt;
+  line-height: normal;
+  font-family: 'Tiempos Headline', sans-serif;
+  position: relative;
+  top: 7rem;
+}
+
+.hero-img-text > p {
+  position: relative;
+  top: 6rem;
+}
+
+.hero-img-button {
+  padding-top: 2rem;
+}
+
+.hero-img-button > button {
+  padding: 10px;
+  background-color: #294464;
+  color: white;
+  font-family: 'Avenir', sans-serif;
+  font-size: 15pt;
+  width: 161%;
```

Comment:
Oh noooooooooooo

Percentages should never go above 100%. You should use a different unit here.